src/pages/en/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,664 @@
+
+## Why nanostores?
+
+The [nanostores](https://github.com/nanostores/nanostores) library allows you to author stores that any component can interact with. We recommend nanostores because:
+- **They're lightweight.** Nanostores ship the bare minimum JS you'll need (less than 1 KB) with zero dependencies.
+- **They're framework-agnostic.** This means sharing state between Preact, Svelte, and Vue should be seemless! Astro is built on flexibility, so we love solutions that offer a similar developer experience no matter your preference.
+
+Still, there are a number of alternatives you can explore. These might include:
+- [Svelte's built-in stores](https://svelte.dev/tutorial/writable-stores)
+- [Sending custom browser events](https://developer.mozilla.org/en-US/docs/Web/Events/Creating_and_triggering_events) between components
+- [Solid signals](https://www.solidjs.com/docs/latest) outside of a component context
+
+:::note
+
+<details>
+<summary>**🙋 How do Svelte stores compare to nanostores?**</summary>
+
+**Nanostores and [Svelte stores](https://svelte.dev/tutorial/writable-stores) are very similar!** In fact, [nanostores allow you to use the same `$` shortcut](https://github.com/nanostores/nanostores#svelte) for subscriptions that you might use with Svelte stores.
+
+If you want to avoid third-party libraries, [Svelte stores](https://svelte.dev/tutorial/writable-stores) are a great cross-island communication tool on their own. Still, you might prefer nanostores if a) you like their add-ons for ["objects"](https://github.com/nanostores/nanostores#maps) and [async state](https://github.com/nanostores/nanostores#lazy-stores), or b) you want to communicate between Svelte and other UI frameworks like Preact or Vue.
+</details>
+
+:::
+
+## Installing nanostores
+
+To get started, install nanostores alongside their helper package for your favorite UI framework:
+
+<UIFrameworkTabs>
+  <Fragment slot="preact">
+  ```shell
+  npm i nanostores @nanostores/preact
+  ```
+  </Fragment>
+  <Fragment slot="react">
+  ```shell
+  npm i nanostores @nanostores/react
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```shell
+  npm i nanostores @nanostores/solid
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+  ```shell
+  npm i nanostores
+  ```
+  :::note
+  No helper package here! Nanostores can be used like standard Svelte stores.
+  :::
+  </Fragment>
+  <Fragment slot="vue">
+  ```shell
+  npm i nanostores @nanostores/vue
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+You can jump into the [nanostores usage guide](https://github.com/nanostores/nanostores#guide) from here, or follow along with our example below!
+
+## Usage example - ecommerce cart flyout
+
+Let's say we're building a simple ecommerce interface with three interactive elements:
+- An "add to cart" submission form
+- A cart flyout to display those added items
+- A cart flyout toggle
+
+<LoopingVideo sources={[{ src: '/videos/stores-example.mp4', type: 'video/mp4' }]} />
```

Comment:
Would be nice to link to a live StackBlitz example here? Some folks prefer to dive right into a live project.

Reply:
Updating our `with-nanostores` template to use this very example! I'll embed a link once that's merged

Reply:
Updated!